### ISSUE TITLE
Improvement: Tool or command to show Town Protected area.

### ISSUE THREAD

**Issue by Wissi** [ldtteam/minecolonies#888]
I believe there is a great demand and really necessary to have a tool or command to show (turn on/off maybe) the Town's protected area of a player or other players in general. Showing maybe an outline in red or the last blocks withing the radius have a ghost block on top (that part is up for the coders to decide?).

So that everyone can know exactly where their Town area is as well as other player's Town area limits.  Definitely synced with the server side for other players.

**Comment by AndrewTru** [comment]
the thing I think this may work is if you copy the outline from the building code and reword it to town limit, yes this change will help other players with remembering the limits, if a block that is a lot like the keep out tape that you use with the building code, just recode it to say city limits.
The only issue I see with it is how it will have to dynamically redo itself every time the limit changes (SSP will not be the issue but SMP will) , when we get to that part that is. I think the limit is fixed at this time, can someone from the team confirm this?

**Comment by marvin-bitterlich** [comment]
@AndrewTru I am against placing blocks to visualize borders, rendering it clientside would be easier imo

**Comment by AndrewTru** [comment]
good point.
